From 20455e987bdc4363dfc9db5c117813c2cdb3f0fd Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 8 Sep 2018 10:08:21 +0200
Subject: [PATCH 1/1] efi_loader: struct efi_simple_text_input_protocol

%s/efi_simple_input_interface/efi_simple_text_input_protocol/

We should be consistent in the naming of the EFI protocol interface
structures. The protocol is called EFI_SIMPLE_TEXT_INPUT_PROTOCOL.

%s/ExtendedVerification/extended_verification/

Use consistent naming of function parameters. Do not use CamelCase.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/serial/serial_efi.c             |  2 +-
 include/efi_api.h                       | 10 +++++-----
 include/efi_loader.h                    |  2 +-
 include/efi_selftest.h                  |  2 +-
 lib/efi_loader/efi_console.c            |  6 +++---
 lib/efi_selftest/efi_selftest_console.c |  2 +-
 6 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/drivers/serial/serial_efi.c b/drivers/serial/serial_efi.c
index 399dfd65fe..1b54d1880f 100644
--- a/drivers/serial/serial_efi.c
+++ b/drivers/serial/serial_efi.c
@@ -17,7 +17,7 @@
 
 /* Information about the efi console */
 struct serial_efi_priv {
-	struct efi_simple_input_interface *con_in;
+	struct efi_simple_text_input_protocol *con_in;
 	struct efi_simple_text_output_protocol *con_out;
 	struct efi_input_key key;
 	bool have_key;
diff --git a/include/efi_api.h b/include/efi_api.h
index 0893a34087..f1c4e59f3a 100644
--- a/include/efi_api.h
+++ b/include/efi_api.h
@@ -308,7 +308,7 @@ struct efi_system_table {
 	u16 *fw_vendor;   /* physical addr of wchar_t vendor string */
 	u32 fw_revision;
 	efi_handle_t con_in_handle;
-	struct efi_simple_input_interface *con_in;
+	struct efi_simple_text_input_protocol *con_in;
 	efi_handle_t con_out_handle;
 	struct efi_simple_text_output_protocol *con_out;
 	efi_handle_t stderr_handle;
@@ -596,11 +596,11 @@ struct efi_input_key {
 	EFI_GUID(0x387477c1, 0x69c7, 0x11d2, \
 		 0x8e, 0x39, 0x0, 0xa0, 0xc9, 0x69, 0x72, 0x3b)
 
-struct efi_simple_input_interface {
-	efi_status_t(EFIAPI *reset)(struct efi_simple_input_interface *this,
-			bool ExtendedVerification);
+struct efi_simple_text_input_protocol {
+	efi_status_t(EFIAPI *reset)(struct efi_simple_text_input_protocol *this,
+				    bool extended_verification);
 	efi_status_t(EFIAPI *read_key_stroke)(
-			struct efi_simple_input_interface *this,
+			struct efi_simple_text_input_protocol *this,
 			struct efi_input_key *key);
 	struct efi_event *wait_for_key;
 };
diff --git a/include/efi_loader.h b/include/efi_loader.h
index 736e82c506..3bf059b447 100644
--- a/include/efi_loader.h
+++ b/include/efi_loader.h
@@ -92,7 +92,7 @@ extern struct efi_runtime_services efi_runtime_services;
 extern struct efi_system_table systab;
 
 extern struct efi_simple_text_output_protocol efi_con_out;
-extern struct efi_simple_input_interface efi_con_in;
+extern struct efi_simple_text_input_protocol efi_con_in;
 extern struct efi_console_control_protocol efi_console_control;
 extern const struct efi_device_path_to_text_protocol efi_device_path_to_text;
 /* implementation of the EFI_DEVICE_PATH_UTILITIES_PROTOCOL */
diff --git a/include/efi_selftest.h b/include/efi_selftest.h
index d0a76d70ca..59b3c080bd 100644
--- a/include/efi_selftest.h
+++ b/include/efi_selftest.h
@@ -53,7 +53,7 @@ enum efi_test_phase {
 };
 
 extern struct efi_simple_text_output_protocol *con_out;
-extern struct efi_simple_input_interface *con_in;
+extern struct efi_simple_text_input_protocol *con_in;
 
 /*
  * Exit the boot services.
diff --git a/lib/efi_loader/efi_console.c b/lib/efi_loader/efi_console.c
index 277eb91e52..426567c946 100644
--- a/lib/efi_loader/efi_console.c
+++ b/lib/efi_loader/efi_console.c
@@ -391,7 +391,7 @@ struct efi_simple_text_output_protocol efi_con_out = {
 };
 
 static efi_status_t EFIAPI efi_cin_reset(
-			struct efi_simple_input_interface *this,
+			struct efi_simple_text_input_protocol *this,
 			bool extended_verification)
 {
 	EFI_ENTRY("%p, %d", this, extended_verification);
@@ -446,7 +446,7 @@ out:
 }
 
 static efi_status_t EFIAPI efi_cin_read_key_stroke(
-			struct efi_simple_input_interface *this,
+			struct efi_simple_text_input_protocol *this,
 			struct efi_input_key *key)
 {
 	struct efi_input_key pressed_key = {
@@ -558,7 +558,7 @@ static efi_status_t EFIAPI efi_cin_read_key_stroke(
 	return EFI_EXIT(EFI_SUCCESS);
 }
 
-struct efi_simple_input_interface efi_con_in = {
+struct efi_simple_text_input_protocol efi_con_in = {
 	.reset = efi_cin_reset,
 	.read_key_stroke = efi_cin_read_key_stroke,
 	.wait_for_key = NULL,
diff --git a/lib/efi_selftest/efi_selftest_console.c b/lib/efi_selftest/efi_selftest_console.c
index eb139c127c..42f51b6520 100644
--- a/lib/efi_selftest/efi_selftest_console.c
+++ b/lib/efi_selftest/efi_selftest_console.c
@@ -9,7 +9,7 @@
 #include <vsprintf.h>
 
 struct efi_simple_text_output_protocol *con_out;
-struct efi_simple_input_interface *con_in;
+struct efi_simple_text_input_protocol *con_in;
 
 /*
  * Print a MAC address to an u16 string
-- 
2.18.0

